src/relay/op/nn/nn.cc

```diff
@@ -1023,7 +1023,8 @@ RELAY_REGISTER_OP("nn.layer_norm")
     .set_attr<FInferCorrectLayout>("FInferCorrectLayout",
                                    NormalizationInferCorrectLayout<LayerNormAttrs>)
     .set_support_level(1)
-    .add_type_rel("LayerNorm", LayerNormRel);
+    .add_type_rel("LayerNorm", LayerNormRel)
+    .set_attr<TOpPattern>("TOpPattern", kInjective);
```

Comment:
Should be kOutFusible? Because it contains reduction part, it can't be fused into other injective ops

Reply:
Done